tools/crowdin/actions/pr-creator/index.js

```diff
@@ -0,0 +1,85 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+(async () => {
+  try {
+    const branch = core.getInput('branch-name');
+    const [owner, repo] = core.getInput('owner-repo').split('/');
+    if (!owner || !repo) {
+      core.setFailed('Must specify a valid ownerName/repoName');
+    }
+    const base = core.getInput('base');
+    const title = core.getInput('title');
+    const body = core.getInput('body');
+    const labelsStr = core.getInput('labels');
+    const labels = labelsStr.trim().split(/,\s+/);
+    const reviewersStr = core.getInput('reviewers');
+    const reviewers = reviewersStr.trim().split(/,\s+/);
+
+    const branchExists = await github.repos
+      .getBranch({
+        owner,
+        repo,
+        branch
+      })
+      .catch(err => {
+        console.info('Branch does not exist. Likely no changes in download?');
+      });
+    if (!branchExists || branchExists.status !== 200) {
+      return;
+    }
+    const pullRequestExists = await github.pulls.list({
+      owner,
+      repo,
+      head: `${repo}:${branch}`
```

Comment:
```suggestion
      head: `${owner}:${branch}`
```

Takes the syntax `nhcarrigan:this-is-my-branch`.